plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php

```diff
@@ -25,161 +25,114 @@ public static function get_all() {
 				'plugins'    => array( 'woocommerce-payfast-gateway' ),
 				'is_visible' => array(
-					(object) array(
-						'type'      => 'base_location_country',
-						'value'     => 'ZA',
-						'operation' => '=',
-					),
+					self::get_rules_for_countries( array( 'ZA', 'GH', 'NG' ) ),
 					self::get_rules_for_cbd( false ),
 				),
+				'category_other' => array( 'ZA', 'GH', 'NG' ),
+				'category_additional' => array(),
 			),
 			array(
 				'id'                      => 'stripe',
 				'title'                   => __( ' Stripe', 'woocommerce' ),
 				'content'                 => __( 'Accept debit and credit cards in 135+ currencies, methods such as Alipay, and one-touch checkout with Apple Pay.', 'woocommerce' ),
-				'image'                   => WC()->plugin_url() . '/assets/images/payment_methods/72x72/stripe.png',
+				'image'                   => WC()->plugin_url() . '/assets/images/stripe.png',
+				'image_72x72'                   => WC()->plugin_url() . '/assets/images/payment_methods/72x72/stripe.png',
 				'plugins'                 => array( 'woocommerce-gateway-stripe' ),
 				'is_visible'              => array(
 					// https://stripe.com/global.
 					self::get_rules_for_countries(
-						array(
-							'AU',
-							'AT',
-							'BE',
-							'BG',
-							'BR',
-							'CA',
-							'CY',
-							'CZ',
-							'DK',
-							'EE',
-							'FI',
-							'FR',
-							'DE',
-							'GR',
-							'HK',
-							'IN',
-							'IE',
-							'IT',
-							'JP',
-							'LV',
-							'LT',
-							'LU',
-							'MY',
-							'MT',
-							'MX',
-							'NL',
-							'NZ',
-							'NO',
-							'PL',
-							'PT',
-							'RO',
-							'SG',
-							'SK',
-							'SI',
-							'ES',
-							'SE',
-							'CH',
-							'GB',
-							'US',
-							'PR',
-						)
+						array( 'AU', 'AT', 'BE', 'BG', 'BR', 'CA', 'CY', 'CZ', 'DK', 'EE', 'FI', 'FR', 'DE', 'GR', 'HK', 'IN', 'IE', 'IT', 'JP', 'LV', 'LT', 'LU', 'MY', 'MT', 'MX', 'NL', 'NZ', 'NO', 'PL', 'PT', 'RO', 'SG', 'SK', 'SI', 'ES', 'SE', 'CH', 'GB', 'US', 'PR', 'HU', 'SL', 'ID', 'MY', 'SI', 'PR' )
 					),
 					self::get_rules_for_cbd( false ),
 				),
+				'category_other' => array( 'AU', 'AT', 'BE', 'BG', 'BR', 'CA', 'CY', 'CZ', 'DK', 'EE', 'FI', 'FR', 'DE', 'GR', 'HK', 'IN', 'IE', 'IT', 'JP', 'LV', 'LT', 'LU', 'MY', 'MT', 'MX', 'NL', 'NZ', 'NO', 'PL', 'PT', 'RO', 'SG', 'SK', 'SI', 'ES', 'SE', 'CH', 'GB', 'US', 'PR', 'HU', 'SL', 'ID', 'MY', 'SI', 'PR' ),
+				'category_additional' => array(),
 				'recommendation_priority' => 3,
 			),
 			array(
 				'id'         => 'paystack',
 				'title'      => __( 'Paystack', 'woocommerce' ),
 				'content'    => __( 'Paystack helps African merchants accept one-time and recurring payments online with a modern, safe, and secure payment gateway.', 'woocommerce' ),
-				'image'      => WC()->plugin_url() . '/assets/images/payment_methods/72x72/paystack.png',
+				'image'      => WC_ADMIN_IMAGES_FOLDER_URL . '/onboarding/paystack.png',
```

Comment:
Just wanted to confirm. Looks like `WC_ADMIN_IMAGES_FOLDER_URL` is the same as `WC()->plugin_url() . '/assets/images`?

Reply:
Ah yes, I think you're right. I'll try to make it consistent by

Reply:
@chihsuan  Do you think using the constant `WC_ADMIN_IMAGES_FOLDER_URL` is better?

Reply:
Changed in [e6458fa](https://github.com/woocommerce/woocommerce/pull/32605/commits/e6458fafe093c89574aaf393c7da24917259481c)

Reply:
Yes, I think it's fewer function calls. I guess we'll rename it to `WC_IMAGES_FOLDER_URL` soon in other PR. 🙂